### PR TITLE
Reorganize hosted and self-hosted cloud pages. Add Cosmic cloud to list of clouds

### PR DESCRIPTION
**Pull request by ddelnano**

### CODE REVIEW

File: content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ metaDescription: "Getting started guide to setup 100% self-hosted Pixie"
 order: 2
 ---
 
-Get Pixie fully managed with [Pixie Community Cloud](/installing-pixie/install-guides/community-cloud-for-pixie) (free forever) or run on your own infrastructure with the following self-managed option.
+Get Pixie fully managed with one of the [Hosted Pixie](/installing-pixie/install-guides/hosted-pixie) or run on your own infrastructure with the following self-managed option.
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie/index.md]: Add `offerings`: "managed with one of the `[Hosted Pixie offerings]`".

File: content/en/02-installing-pixie/03-install-guides/index.md [hunk 2]
```diff
@@ -9,28 +9,23 @@ redirect_from:
...
+If you want to deploy Pixie without needing to host your own version of Pixie Cloud, select one of the Cloud offerings on the [Hosted Pixie](/installing-pixie/install-guides/hosted-pixie) page and following the install instructions.
 
-With Community Cloud for Pixie, telemetry data is exclusively stored on your Kubernetes cluster (not in Pixie Cloud).
+When using a hosted Pixie backend, telemetry data is exclusively stored on your Kubernetes cluster (not in Pixie Cloud).
 
 ## Self-Hosted Pixie
 
 Pixie also supports for self-hosting of the entire platform, including Pixie Cloud. Follow the [Self-Hosted Pixie Guide](/installing-pixie/install-guides/self-hosted-pixie) for the steps.
 
-## Air Gapped Pixie
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/index.md:30]: Did you intend to remove air-gapped Pixie?

File: content/en/02-installing-pixie/04-install-schemes/01-cli.md [hunk 3]
```diff
@@ -108,12 +108,15 @@ rpm -i pixie-px.x86_64.rpm
 px deploy --help
 
 # Deploy the Pixie Platform in your K8s cluster (No OLM present on cluster).
+export PL_CLOUD_ADDR=<getcosmic.ai or withpixie.ai>
```
Comment (aimichelle) [inline content/en/02-installing-pixie/04-install-schemes/01-cli.md:111]: Ideally the dropdown would update these exports.

**Review by aimichelle — APPROVED** [review]

File: content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie/index.md (hunk 1 above)
Comment (ddelnano) [reply content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie/index.md]: Fixed in latest commit.

**Pull request merged**